view_pdf_file/lib/constants.dart

```diff
@@ -0,0 +1,10 @@
+class Constants {
+  static final String pdfURL =
+      "https://raw.githubusercontent.com/nisrulz/flutter-examples/master/view_pdf_file/assets/Hello.pdf";
+
+  static final String pdfURLdevelop =
+      "https://raw.githubusercontent.com/nisrulz/flutter-examples/develop/view_pdf_file/assets/Hello.pdf";
```

Comment:
To test if the url works, I changed this url to 

```
  static final String pdfURLdevelop =
      "https://raw.githubusercontent.com/tsvillain/flutter-examples/develop/view_pdf_file/assets/Hello.pdf";
```

So in this case, I think your URL is correct. Because once it gets merged, it will be under `nisrulz` user and `master` branch.

Reply:
I would say you delete this URL also.